tests/test_casbin.rs

```diff
@@ -0,0 +1,22 @@
+use actix_casbin::{CasbinActor, CasbinCmd};
+use casbin::prelude::*;
+
+#[actix_rt::test]
+async fn test_enforcer() {
+    let m = DefaultModel::from_file("examples/rbac_model.conf")
+        .await
+        .unwrap();
+    let a = FileAdapter::new("examples/rbac_policy.csv");
+    let addr = CasbinActor::new(m, a).await;
```

Comment:
you can create it directly `CasbinActor::new("examples/rbac_model.conf", "examples/rbac_policy.csv").await

Reply:
could you also make a multithread test?